asllib/doc/ASLmacros.tex

```diff
@@ -215,6 +215,7 @@
 \newcommand\cardinality[1]{\hyperlink{def-cardinality}{|}#1\hyperlink{def-cardinality}{|}}
 \newcommand\filterlist[0]{\hyperlink{def-filterlist}{\texttt{filter\_list}}}
 \newcommand\unziplist[0]{\hyperlink{def-unziplist}{\texttt{unzip}}}
+\newcommand\uniquelist[0]{\hyperlink{def-uniquelist}{\texttt{unique}}}
```

Comment:
please use \textfunc instead of \texttt. Actually, same for the functions above.